openslides_backend/action/actions/motion/json_upload.py

```diff
@@ -0,0 +1,708 @@
+                    "id": id_,
+                }
+            elif check_result == ResultType.NOT_FOUND or id_ == 0:
+                self.row_state = ImportState.NEW
+                entry["number"] = {
+                    "value": number,
+                    "info": ImportState.DONE,
+                }
+            elif check_result == ResultType.FOUND_MORE_IDS:
+                self.row_state = ImportState.ERROR
+                entry["number"] = {
+                    "value": number,
+                    "info": ImportState.ERROR,
+                }
+                messages.append("Error: Found multiple motions with the same number")
+        else:
+            category_id: Optional[int] = None
+            if entry.get("category_name"):
+                category_id = entry["category_name"].get("id")
+            self.row_state = ImportState.NEW
+            value: Dict[str, Any] = {}
+            self.set_number(
+                value,
+                meeting_id,
+                self._get_first_workflow_state_id(meeting_id),
+                None,
+                category_id,
+                other_forbidden_numbers=self._previous_numbers,
+            )
+            if number := value.get("number"):
+                entry["number"] = {"value": number, "info": ImportState.GENERATED}
+                self._previous_numbers.append(number)
+
+        has_submitter_error: bool = False
+        for fieldname in ["submitter", "supporter"]:
+            if users := entry.get(f"{fieldname}s_username"):
+                if not isinstance(users, list):
+                    users = [users]
+                verbose = entry.get(f"{fieldname}s_verbose", [])
+                if not isinstance(verbose, list):
+                    verbose = [verbose]
+                verbose_user_mismatch = len(verbose) > len(users)
+                username_set: Set[str] = set([])
+                entry_list: list[Dict[str, Any]] = []
+                message_set = set()
+                lookup = (
+                    self.submitter_lookup
+                    if fieldname == "submitter"
+                    else self.supporter_lookup
+                )
+                for user in users:
+                    if isinstance(user, str):
+                        if verbose_user_mismatch:
+                            entry_list.append(
+                                {"value": user, "info": ImportState.ERROR}
+                            )
+                        elif user in username_set:
+                            entry_list.append(
+                                {"value": user, "info": ImportState.WARNING}
+                            )
+                            message_set.add(
+                                f"At least one {fieldname} has been referenced multiple times"
+                            )
+                        else:
+                            username_set.add(user)
+                            found_users = lookup.get(user, [])
```

Comment:
You cannot get more than one user, the username is unique. If you find duplicate users, you should throw an exception and finish the json_upload immediately,

Reply:
Done